crates/polars/src/docs/eager.rs

```diff
@@ -120,7 +124,7 @@
 //! let multiplied = ca_int * 2.0;
 //! let multiplied = s_flt * 2.0;
 //!
-//! // or broadcast Series to match the operands type
+//! // or broadcast ['Series'] to match the operands type
```

Comment:
That don't look right :)

Reply:
Good catch. Reverted in:
https://github.com/pola-rs/polars/pull/11007/commits/1bfa69421f1a57fca102694940d9642f72793ba3#diff-a976127b0782570471eeed4567ed71a313680122e0ae84a1c1ebafa3147c230a